src/wp-includes/block-editor.php

```diff
@@ -325,7 +325,11 @@ function _wp_get_iframed_editor_assets() {
 
 	foreach ( $block_registry->get_all_registered() as $block_type ) {
 		if ( ! empty( $block_type->style ) ) {
-			$style_handles[] = $block_type->style;
+			if ( is_array( $block_type->style ) ) {
+				$style_handles[] = $block_type->style[0];
```

Comment:
Same questions as on the GB PR - why only use the first one?

Reply:
Without this PR, `_wp_multiple_block_styles` replaces the `style` property from an array to the first element of that array. With this PR, the `style` property remains an array but we still want to only use the first element of it. That's what happens here.

Reply:
Right, I wonder if we can make it a bit smarter. Maybe we can add all the strings and filter out all the arrays?

Reply:
If we filter out all the arrays we won't be able to use the information from these arrays. That's the pickle we're in: Updating `style` to accept arrays means we have to deal with arrays now.

Reply:
When I say filter them out I mean simply don't set them on $style_handles.

Like this:

```
if ( is_array( $block_type->style ) ) {
	foreach( $block_type->style as $single_style) {
		if ( ! is_array( $single_style) ) {
			$style_handles[] = $single_style;
		}
	}
}
```

Reply:
Gotcha! I just did that.

Reply:
Nice. Do we need to do the same in the Gutenberg PR?